ccmlib/scylla_repository.py

```diff
@@ -318,7 +328,7 @@ def setup(version, verbose=True, skip_downloads=False):
                     scylla_package=os.path.join(s3_url, scylla_package_path)
                 )
 
-        version = os.path.join(*type_n_version)
+        version = os.path.join(*type_n_version) + "-debug" * scylla_debug
```

Comment:
here too, `* scylla_debug` is too sophisticated :)
`f"{'-debug' if scylla_debug else ''}"` is longer, but clearer.
Maybe we can store it in a variable and reuse it multiple times.
In theory, it could even be the value of `scylla_debug` instead of a boolean, but it's a hack...

Reply:
LGTM otherwise